packages/ketcher-core/src/application/editor/modes/SequenceMode.ts

```diff
@@ -937,11 +941,19 @@ export class SequenceMode extends BaseMode {
   private getNewSequenceItemPosition(
     previousNode?: SubChainNode,
     nodeBeforePreviousNode?: SubChainNode,
+    currentNode?: SubChainNode,
+    nextNode?: SubChainNode,
   ) {
     const offsetFromPrevious = new Vec2(1, 1);
 
     if (previousNode && !(previousNode instanceof EmptySequenceNode)) {
       return previousNode.lastMonomerInNode.position.add(offsetFromPrevious);
+    } else if (
+      currentNode &&
+      nextNode &&
+      previousNode instanceof EmptySequenceNode
+    ) {
+      return currentNode.lastMonomerInNode.position.add(new Vec2(0, 0));
```

Comment:
I believe it could be better to use firstMonomerInNode insterad of lastMonomerInNode

Reply:
And not sure that we need to add Vec(0,0)